### ISSUE TITLE
[FEATURE] Please add ability to change superadmin only use to operator or admin use.

### ISSUE THREAD

**Issue by Psycho-Knight** [thebigsleepjoe/TTT-Bots-2#58]
I would like my server operators (friends) to be able to play with bots while im off the server

**Comment by thebigsleepjoe** [comment]
When you say operators, are you referring to ULX by chance?

In the meantime, you can set the bot quota with `!botquota` in chat, or do `ttt_bot_quota` to force a certain number of bots in the server, and it works between map changes.

**Comment by Psycho-Knight** [comment]
I mean in the sense that as of right now superadmins are the only ones with
acces to use ttt_bot_add x and I would like to change that to operators,
admins, and owners to be able to use ttt_bot_add x

**Comment by thebigsleepjoe** [comment]
Can you clarify what you mean by "operator" or "admin"? Are you using the ULX definition?

**Comment by Psycho-Knight** [comment]
Yes, ulx groups and custom made groups, how can I edit the lua code to
allow more groups other than superadmins to use it
